benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.benchto.integrationtest;
+
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import io.trino.benchto.driver.BenchmarkProperties;
+import io.trino.benchto.driver.DriverApp;
+import io.trino.benchto.driver.execution.ExecutionDriver;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.context.ApplicationContext;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.util.ReflectionTestUtils;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponents;
+import org.springframework.web.util.UriComponentsBuilder;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static com.google.common.util.concurrent.Uninterruptibles.sleepUninterruptibly;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = DriverApp.class)
+@org.springframework.boot.test.IntegrationTest("executionSequenceId=BEN_SEQ_ID")
+public class BenchtoTrinoIntegrationTest
+{
+    static PostgreSQLContainer<?> postgres;
+    static GenericContainer<?> service;
+    static GenericContainer<?> trino;
+
+    @Autowired
+    private ExecutionDriver executionDriver;
+
+    @Autowired
+    private BenchmarkProperties benchmarkProperties;
+
+    @Autowired
+    protected ApplicationContext context;
+
+    @BeforeClass
+    public static void setup()
+    {
+        Network network = Network.newNetwork();
```

Comment:
will this cleanup automatically after test is finished?

Reply:
Yes, I see no networks left after tests execution.